pmm-app/docker-compose.yaml

```diff
@@ -2,7 +2,7 @@ version: "3"
 services:
   pmm-server:
     container_name: pmm-server
-    image: perconalab/pmm-server:dev-latest
+    image: perconalab/pmm-server-fb:PR-1067-10053da
```

Comment:
Should be noop here. Pls use docker-compose.local.yml

Reply:
oh, sorry, committed accidentally